dub.sdl

```diff
@@ -12,3 +12,6 @@ configuration "dip1000" {
        dflags "-dip1000"
        versions "DIP1000"
 }
+configuration "preview_in" {
+	dflags "-preview=in"
+}
```

Comment:
Unless this is used in the CI, this is not a good idea. The way to pass previews is to pass them from the application recipe file via the dependencies' dflags.

Reply:
Does setting the dflags in the main project make it to the dependencies? My point was that you can use this configuration to turn on preview=in for your dependency.

I know that setting the environment DFLAGS goes everywhere, but I thought dflags was specific to the project itself.

Reply:
So this leaves a lot to be desired. If I specify "dflags" in my application, I get the flag there, but not on the dependencies. I have to specify the flags in the dependency (which means a more complex dependency specification), then I get it on the dependency, but then I have to specify on *every* dependency, and their subdependencies. Plus I have to specify in the application itself.

But yes, I agree these configs aren't really proper. We need a better way to say "for all compilations, use these flags".

Reply:
>  but then I have to specify on every dependency, and their subdependencies. 

No, if you specify it on a dependency, it is transitive. The rest is correct. I agree the current situation is suboptimal but that was the only way we could get things to work without changing dub's behavior.